src/Dfc.CourseDirectory.Core/CsvUtil.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.IO;
+
+namespace Dfc.CourseDirectory.Core
+{
+    public static class CsvUtil
```

Comment:
Classes called util attract other stuff, how about:
```suggestion
    public static class CsvLineCounter
```

Reply:
I have another change Coming Soon (TM) which adds something else here so the naming will make more sense then.